Solutions/vArmour/Data Connectors/Connector_vArmour_AppController_CEF.json

```diff
@@ -0,0 +1,141 @@
+{
+    "id": "vArmourAC",
+    "title": " vArmour Application Controller",
+    "publisher": "vArmour",
+    "descriptionMarkdown": "Common Event Format (CEF) is an industry standard format on top of Syslog messages, used by many security vendors to allow event interoperability among different platforms. By connecting your CEF logs to Azure Sentinel, you can take advantage of search & correlation, alerting, & threat intelligence enrichment for each log.",
```

Comment:
This description only talks about CEF datatype, but should cover meaningful explanation of what data the data connector enables bringing in, in what format and pointer to appliance documentation as applies